eodag/api/product/metadata_mapping.py

```diff
@@ -610,6 +610,18 @@ def convert_dict_filter(
                 result[key] = match.value
             return result
 
+        @staticmethod
+        def convert_dict_filter_and_sub(
+            input_dict: dict[Any, Any], args: str
+        ) -> Union[dict[Any, Any], list[Any]]:
+            """Fitlers dict items using jsonpath and then apply recursive_sub_str"""
+            jsonpath_filter_str, old, new = args.split("|")
```

Comment:
for args "split", please use the same mechanism as in other methods (comma-separated, `ast.literal_eval`)